### PR TITLE
finish sign up (2)

### PR DESCRIPTION
**Pull request by sangpt**

### CODE REVIEW

**Comment by sangpt** [comment]
------------------------------------------
[o] Build Success!
------------------------------------------

[!] rspec: failed but ignored
[o] rails_best_practices: success
[o] brakeman: success
[o] reek: success
[o] scss-lint: success
[o] eslint: success
[o] bundle-audit: success
[o] rubocop: success

**Comment by sangpt** [comment]
ready

File: app/views/devise/registrations/edit.html.erb [hunk 1]
```diff
@@ -0,0 +1,48 @@
+<h2><%= t ".title" %> <%= resource_name.to_s.humanize %></h2>
+
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for resource, as: resource_name, url: registration_path(resource_name),
+      html: { method: :put } do |f| %>
```
Comment (tuyenvd1177) [inline app/views/devise/registrations/edit.html.erb]: remove space

**Comment by tuyenvd1177** [comment]
commented

**Comment by sangpt** [comment]
ready

**Pull request merged**